static/css/inbox.less

```diff
@@ -1409,3 +1409,7 @@ a.fa:hover {
     top: 55px;
   }
 }
+
+.material .card:last-child {
+  margin-bottom: 0;
+}
```

Comment:
Move this up to around line 10 so it looks something like this:

```css
.material {
  .card {
    border: 1px solid @seperator-color;
    background-color: white;
    margin-bottom: 16px;
    border-radius: @radius-size-small;
    padding-top: 0;
    padding-bottom: 0;
    clear: both;
    &:last-child {
      margin-bottom: 0;
    }
    a.cell {
...
```

This puts it near the style it's overriding so you can see everything in one place.